### DISCUSSION TITLE
.htaccess file for LiteSpeed server

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#3075] [Show and tell]
I host my log on LiteSpeed server. Since default .htaccess file doesn't work on LiteSpeed servers, here is working code:

************
Options -Indexes
ServerSignature off

RewriteEngine On

RewriteCond %{REQUEST_URI} ^/backup/$
RewriteRule ^(.*)$ /index.php?/$1 [QSA]

RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^(.*)$ /index.php?/$1 [L]

RedirectMatch 404 ^/\.git`
************

73, Dragan 4O4A

**Comment by magicbug** [comment]
Thanks for sharing @4O4A I'm sure others will find it helpful
